app/views/fragments/head.scala.html

```diff
@@ -24,4 +24,4 @@
     <link rel="shortcut icon" type="image/png" href="@routes.Assets.at("images/favicons/32x32.ico")"/>
 }
 
-@* <script src="//cdn.optimizely.com/js/293455139.js"></script> *@
+<script src="//cdn.optimizely.com/js/293455139.js"></script>
```

Comment:
As this is in the head (I get why A/B testing scripts are in the head) you should definitely add `async defer` attributes to this as in the currently could cause SPOF errors. See https://github.com/guardian/membership-frontend/pull/421